airflow/www/static/feed.js

```diff
@@ -0,0 +1,81 @@
+/**
```

Comment:
should it be # instead of /** ?

Reply:
```# -*- coding: utf-8 -*-
#
# Licensed to the Apache Software Foundation (ASF) under one
# or more contributor license agreements.  See the NOTICE file
# distributed with this work for additional information
# regarding copyright ownership.  The ASF licenses this file
# to you under the Apache License, Version 2.0 (the
# "License"); you may not use this file except in compliance
# with the License.  You may obtain a copy of the License at
#
#   http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing,
# software distributed under the License is distributed on an
# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
# KIND, either express or implied.  See the License for the
# specific language governing permissions and limitations
# under the License.
```

Reply:
Isn't that Python syntax, though? This is what I was looking at: https://github.com/lyft/incubator-airflow/blob/9d68010b98138098f511c03cb2534ff7b584ca74/airflow/www/static/mode-sql.js#L1

Reply:
oh you are right...

Reply:
I ran the script locally and it doesn't flag this file. Looks like the error might be different now:

```
tar (child): /home/travis/.travis_cache//hive/hive.tar.gz: Cannot open: Permission denied
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now
Failed twice in downloading and unpacking hive!
ERROR: InvocationError for command '/home/travis/build/lyft/incubator-airflow/scripts/ci/setup_env.sh' (exited with code 1)
```